client/my-sites/stats/feedback/index.tsx

```diff
@@ -101,6 +104,10 @@ function StatsFeedbackController( { siteId }: FeedbackProps ) {
 		if ( action === FEEDBACK_ACTION_DISMISS_FLOATING_PANEL ) {
 			setIsFloatingPanelOpen( false );
 		}
+		if ( action === FEEDBACK_ACTION_LEAVE_REVIEW ) {
+			setIsFloatingPanelOpen( false );
+			window.open( FEEDBACK_LEAVE_REVIEW_URL );
```

Comment:
```suggestion
			window.open( FEEDBACK_LEAVE_REVIEW_URL, '_blank', 'noopener, noreferrer' );
```
I recommend enhancing the security when opening the new tab.

Reply:
The button is more of an external link and we should use anchors wherever we could to avoid being blocked by browsers. Happy to be corrected tho.

Reply:
> The button is more of an external link and we should use anchors wherever we could to avoid being blocked by browsers.

I don't quite understand what you are saying here, @kangzj. Can you elaborate?

Reply:
> wherever we could to avoid being blocked by browsers

I remember that happened sometimes. Is it a security feature from browsers?

Reply:
> I don't quite understand what you are saying here

[The reference](https://stackoverflow.com/questions/14643040/allow-window-open-to-open-new-window-and-not-popup) may be helpful, @a8ck3n.

Reply:
<img width="236" alt="image" src="https://github.com/user-attachments/assets/99bfe08e-5a5b-4d88-86fe-9ee7dd2ebc1c">

This looks like a button, but it works as a link, so we'd better use an anchor. The benefit of that than `window.open` is that, sometimes, the latter got blocked by the browser. 

Even with a link, we should still be able to trigger actions to dismiss the popup via `onClick`. Does that make sense?

Reply:
I could be wrong - feel free to ship and see if that's still a thing after 😄

Reply:
From my testing, opening a new tab automatically in the `useEffect` would be blocked, but our behavior will not be blocked since we had an action to trigger it. 🙂

Reply:
> we should still be able to trigger actions to dismiss the popup

Is the popup you mean the new tab for review or the feedback floating panel?

Reply:
Thanks for the link @dognose24. So the goal is to avoid being blocked by popup blockers. The discussion there notes we should be fine as long as we are responding to a user event. The example they give is with an HTML anchor tag's `click` event. We should be okay as we're doing this via the `onClick` handler of a button, which is in direct response to a user action.

Just to be safe, I tested this in Chrome and Firefox and it worked correctly in both. I then installed uBlock Origin and turned on the strict popup blocker and it still worked correctly for me.

My conclusion is that it isn't necessary to use anchor tags for this and `onClick` handlers should be okay.

Reply:
Yes, let's see if there will be some report about it.

Reply:
> the feedback floating panel?

The panel is not seen as a popup because it doesn't have it's own window context. That should be fine.

Reply:
Follow-up; changing the URL and testing again caused the window to be blocked in uOrigin but only if the strict popup blocking feature was enabled. The default settings did not block us. The user would have to both have uBlock origin installed and have updated their settings for windowing event to be blocked.

Reply:
> I recommend enhancing the security when opening the new tab.

My understanding is that this is no longer needed. The `window.open()` function passes `_blank` by default. All modern browsers from 2021 onward automatically treat this the same as having `noopener` and `noreferrer` set. [reference](https://mathiasbynens.github.io/rel-noopener/)

Takeaway: Calling `window.open( 'MY_URL' )` is the same as providing those values.

Of note, our linter rules are out of date and flag this on anchor tags.

Reply:
Nice sleuthing @a8ck3n ! TIL